src/components/QuranReader/TranslationView/TranslationText/TranslationAndReference.tsx

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+import React, { MouseEvent } from 'react';
+
+import classNames from 'classnames';
+import Link from 'next/link';
+
+import styles from './TranslationText.module.scss';
+
+import { logButtonClick } from '@/utils/eventLogger';
+import { Direction, toLocalizedVerseKey } from '@/utils/locale';
+import { getChapterWithStartingVerseUrl } from '@/utils/navigation';
+
+interface Props {
+  text: string;
+  fontClass: string;
+  direction: string;
+  onTextClicked: (event: MouseEvent) => void;
+  shouldShowReference?: boolean;
+  chapterName?: string;
+  reference?: string;
+  lang: string;
+}
+
+const TranslationAndReference: React.FC<Props> = ({
+  text,
+  fontClass,
+  direction,
+  onTextClicked,
+  shouldShowReference = false,
+  chapterName,
+  reference,
+  lang,
+}) => {
+  return (
+    <div className={classNames(styles.text, fontClass, shouldShowReference && styles.textCenter)}>
```

Comment:
`styles[langData.direction]` that used to exist inside `src/components/QuranReader/TranslationView/TranslationText/index.tsx` is no longer being used. Is this intentional?